src/Lykke.Job.BlockchainMonitoring.AzureRepositories/LastFinishedCashoutMoment/LastCashoutMomentRepository.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AzureStorage;
+using AzureStorage.Tables;
+using Common;
+using Lykke.Common.Log;
+using Lykke.Job.BlockchainMonitoring.Domain.Repositories;
+using Lykke.SettingsReader;
+
+namespace Lykke.Job.BlockchainMonitoring.AzureRepositories.LastFinishedCashoutMoment
+{
+    public class LastCashoutMomentRepository: ILastCashoutMomentRepository
+    {
+        private readonly INoSQLTableStorage<LastCashoutMomentEntity> _storage;
+
+        private LastCashoutMomentRepository(INoSQLTableStorage<LastCashoutMomentEntity> storage)
+        {
+            _storage = storage;
+        }
+
+        public static ILastCashoutMomentRepository Create(
+            IReloadingManager<string> connectionString,
+            ILogFactory log)
+        {
+            var storage = AzureTableStorage<LastCashoutMomentEntity>.Create(
+                connectionString,
+                "MonitoringLastCashoutMoments",
+                log);
+
+            return new LastCashoutMomentRepository(storage);
+        }
+        
+        public async Task<IEnumerable<(string assetId, string assetMetricId, DateTime moment, Guid operationId)>> GetAllAsync()
+        {
+            return (await _storage.GetDataAsync()).Select(p => (p.AssetId, p.AssetMetricId, p.Moment, p.OperationId));
```

Comment:
Full scan. Is it ok here?

Reply:
Yes because there are minimal count of assets (10 for now).